From bec9817c932f93f1641297c8fc7454bb5dfc0a43 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Wed, 15 Oct 2025 13:59:24 +0200
Subject: [PATCH] Inherit from Adwaita theme

Adding "Adwaita" icon theme to "Inherits" fixes some missing symbolic icons
in GNOME (GTK4) applications (e.g. sidebar in Nautilus), while adding
"AdwaitaLegacy" before that causes the non-symbolic variants are used
instead of symbolic ones unless it's explicitly requested (e.g. rotation
and mirror icons in GPicView).
---
 nuoveXT2/index.theme | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nuoveXT2/index.theme b/nuoveXT2/index.theme
index 6726e22..6bc0ff9 100644
--- a/nuoveXT2/index.theme
+++ b/nuoveXT2/index.theme
@@ -1,7 +1,7 @@
 [Icon Theme]
 Name=nuoveXT.2.2
 Comment=nuoveXT 2.2 Icon Theme for LXDE
-Inherits=nuoveXT-1.6,Tango,gnome,crystalsvg
+Inherits=nuoveXT-1.6,Tango,AdwaitaLegacy,Adwaita,gnome,crystalsvg
 Example=x-directory-normal
 
 # KDE Specific Stuff
